docs/source/conf-html.py

```diff
@@ -39,3 +39,10 @@
     # Ensure the nav bar always stays on top of page.
     'navbar_fixed_top': "false",
 }
+
+# -- Fix up angstrom symbol for mathjax
+rst_prolog = r"""
+
+:math:`\renewcommand\AA{\unicode{x212B}}`
+
+"""
```

Comment:
Should there be a newline here? I also wonder if this is really windows only? We see a blank documentation page also on linux in the last nightly.